test/clubbing_top_level.txt

```diff
@@ -0,0 +1 @@
+<table border="1"><tr><th>aaa</th><th>ccc</th></tr><tr><td>1</td><td>2</td></tr><tr><td>bbb</td><td>2</td></tr></table>
```

Comment:
For consistency, IMHO, it should behave as a list.

```
>>> json2html.convert([{"a": 1}])
```
returns
```
<ul><li><table border="1"><tr><th>a</th><td>1</td></tr></table></li></ul>
```

Also, I think it would be very helpful for a user to opt for `cubbing`. Be default, we can turn on the clubbing logic but user's preference (in form of passing an argument, similar to table_attributes), should override it. What's your thought?